src/ui/order/OrderList.tsx

```diff
@@ -0,0 +1,164 @@
+import { SyntheticEvent, useState } from "react";
+import {
+  BooleanField,
+  Count,
+  CreateButton,
+  DatagridConfigurable,
+  DatagridConfigurableProps,
+  DateField,
+  ExportButton,
+  FunctionField,
+  List,
+  NumberField,
+  ReferenceField,
+  SelectColumnsButton,
+  TextField,
+  TopToolbar,
+} from "react-admin";
+import Tabs from "@mui/material/Tabs";
+import Tab from "@mui/material/Tab";
+import Box from "@mui/material/Box";
+import { AddressField } from "../customField";
+import { Order } from "../../types";
+
+interface TabPanelProps {
+  children?: React.ReactNode;
+  index: number;
+  value: number;
+}
+
+const a11yProps = (index: number) => {
+  return {
+    id: `simple-tab-${index}`,
+    "aria-controls": `simple-tabpanel-${index}`,
+  };
+};
+
+const TabPanel = (props: TabPanelProps) => {
+  const { children, value, index, ...other } = props;
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`simple-tabpanel-${index}`}
+      aria-labelledby={`simple-tab-${index}`}
+      {...other}
+    >
+      {value === index && <Box sx={{ p: 3 }}>{children}</Box>}
+    </div>
+  );
+};
+
+const tabs = ["ordered", "delivered", "cancelled"];
+
+const defaultOmits = [
+  "basket",
+  "total_ex_taxes",
+  "delivery_fees",
+  "tax_rate",
+  "taxes",
+  "returned",
+];
+
+export const OrderList = () => {
+  const [value, setValue] = useState(0);
+  const filter = { status: tabs[value] };
+
+  const handleChange = (event: SyntheticEvent, newValue: number) => {
+    setValue(newValue);
+  };
```

Comment:
There's another option to implement this tabed datagrid that leverage the ListContext. But this works too!